library/src/main/java/kwasm/ast/Limit.kt

```diff
@@ -14,10 +14,9 @@
 
 package kwasm.ast
 
-import kwasm.format.text.IntegerLiteral
 import kwasm.format.text.Type.Limits
 
 /**
  * This is a data class that represents the limits of a [Limits] object.
  */
-data class Limit(val min: IntegerLiteral.Unsigned, val max: IntegerLiteral.Unsigned)
\ No newline at end of file
+data class Limit(val min: ULong, val max: ULong)
```

Comment:
Revert this file to origin/master, my PR covered these changes but will definitely cause merge conflicts.

Reply:
Done